### ISSUE TITLE
Unable display attachment preview when attachment is a svg file

### ISSUE THREAD

**Issue by baolanlequang** [ComPlat/chemotion_ELN#1187]
- I am a developer

# Expected Behavior

Upload a `.svg` file and see the preview image

# Current Behavior

Preview image is unavailable

## Steps to Reproduce

Please watch the video

https://user-images.githubusercontent.com/9006482/224151007-2e74e62b-dbf6-4347-aa37-018287410054.mp4

## Context

 - OS: Ubuntu
 - Chrome Browser 
 - Firefox Browser
 - ELN branch `main`

## Failure Logs / Screenshots

The third-party `Thumbnailer` create `nil` value for svg file. The code of this is at `app/usecases/attachments/thumbnail/thumbnail_creator.rb`

```ruby
def create_derivative(tmp_path, _, _, result, record)
    begin
        thumbnail = Thumbnailer.create(tmp_path)
        add_thumbnail_to_record(thumbnail, record, result) if thumbnail.present?
    rescue StandardError => e
        Rails.logger.info('**** ThumbnailCreator failed ***')
        Rails.logger.error e
    end

    result
end
```

**Comment by FabianMauz** [comment]
I think, the problem seems to occur a little bit earlier. With the change to MIME type instead of file ending the Thumbailer and Annotation derivatives are not created because in:
attachment_uploader.rb line 44 the resulting type for a svg file is svg+xml.
A quick hotfix would be to add the following snibbet after line 44:

file_extension = '.svg' if file_extension == '.svg+xml'

For a more robust

**Comment by PiTrem** [comment]
> file_extension = '.svg' if file_extension == '.svg+xml'

Thx @FabianMauz 

added to https://github.com/ComPlat/chemotion_ELN/commit/87af59e0597fd97634f27f3fe62008f4f34e68b3

**Comment by baolanlequang** [comment]
Closed due to this was fixed at v1.5.0
